create-aliquot.avsc

```diff
@@ -0,0 +1,110 @@
+{
+    "type": "record",
+    "name": "CreateAliquotInMLWH",
+    "namespace": "uk.ac.sanger.psd",
+    "doc": "Create an Aliquot in MLWH coming from Traction.",
+    "fields": [
+        {
+            "name": "messageUuid",
+            "type": {
+                "type": "string",
+                "logicalType": "uuid"
+            },
+            "doc": "Unique message ID."
+        },
+        {
+            "name": "messageCreateDateUtc",
+            "type": {
+                "type": "long",
+                "logicalType": "timestamp-millis"
+            },
+            "doc": "Date (UTC) that the message was created."
+        },
+        {
+            "name": "id_aliquot_lims",
+            "type": "string",
+            "doc": "The LIMS system that the aliquot was created in"
```

Comment:
Would it be more intuitive to have the doc as ' The id of the aliquot that was created in'?